scalableminds/webknossos
# use position + zoom from dataset defaultconfiguration for thumbnail

### Steps to test:
- insert custom default configuration for a dataset (see #2451)
- refresh thumbnail, should show selected position

### Issues:
- fixes #2453

------
